### ISSUE TITLE
Running over someone in the ATST should count as a kill for the driver

### ISSUE THREAD

**Issue by deathsythe47** [TheSil/base_enhanced#59]
This is definitely a bug, because with other vehicles (swoop) you can ram somebody and it counts as a kill.

Currently if player A runs over player B, it just says "player B died" and subtracts 1 point from player B.

Instead it should say "player B was killed by player A" and add 1 point to player A.

**Comment by deathsythe47** [comment]
fixed in d3ea1e01ce909f87774a08c5ad938e628136d226

**Comment by deathsythe47** [comment]
Here is fix c7a24ad1d3248c66b95e7ddaac2b00e795ac011f

```
G_Damage(under, (gentity_t *)ent->m_pVehicle->m_pPilot, (gentity_t *)ent->m_pVehicle->m_pPilot, down, under->r.currentOrigin, 100, 0, MOD_CRUSH);
```

**Comment by TheSil** [comment]
Used your approach, just with slight modification. See following commit.
